torch/autograd/__init__.py

```diff
@@ -333,7 +333,7 @@ def variable(*args, **kwargs):
                                 _supported_activities, _add_metadata_json, SavedTensor,
                                 _push_saved_tensors_default_hooks, _pop_saved_tensors_default_hooks)
 
-from torch._C._autograd import (_ProfilerResult, _KinetoEvent, _kineto_step,
+from torch._C._autograd import (_ProfilerResult, _KinetoEvent, _ProfilerEvent, _kineto_step,
```

Comment:
nit: as much as possible we want to limit the ties between profiler and the autograd namespace. (As it is only associated for legacy reasons.) Better to put this in `torch.profiler`

Reply:
Yes, I'll get rid of that, I no longer needed that symbol for type checking I think.